project/PATCH-RELEASES.md

```diff
@@ -0,0 +1,53 @@
+# Docker patch (bugfix) release process
+
+Patch releases (the 'Z' in vX.Y.Z) are intended to fix major issues in a
+release. Docker open source projects follow these procedures when creating a
+patch release;
+
+After each release (both "major" (vX.Y.0) and "patch" releases (vX.Y.Z)), a
+patch-release milestone (vX.Y.Z + 1) is created.
```

Comment:
I still think it can be confusing because people will think there will be such a patch release. I would prefer naming the milestone differently. `vX.Y.Z+1-maybe` ? `patch` ? Or revive `next` and actually use it correctly?